### PR TITLE
Add templates to insert function

### PR DESCRIPTION
**Pull request by gayanDeegala**

### CODE REVIEW

File: src/components/JsonEditor/JsonEditor.tsx [hunk 1]
```diff
@@ -2,16 +2,39 @@ import React, { useEffect } from 'react';
...
+        getAllNodes().map((node: any) => {
+            const temp = {
+                text: extractField(node.key),
+                title: node.node.description,
+                className: 'jsoneditor-type-object',
+                field: extractField(node.key),
+                value: node.data,
+                key: node.key
+            }
+            templates.push(temp);
```
Comment (malindakpt) [inline src/components/JsonEditor/JsonEditor.tsx:36]: Get the output of the map function and set it to props you need

File: src/components/JsonEditor/JsonEditor.tsx [hunk 2]
```diff
@@ -2,16 +2,39 @@ import React, { useEffect } from 'react';
 import classes from './JsonEditor.module.scss'
 import mauiA from '../../config/MauiA.json';
 import { JsonEditorContainer } from "../JsonEditorContainer/JsonEditorContainer";
-import { addNode, getAllNodes, loadSchema, removeNode } from '../../validator/Validator';
+import { getAllNodes, loadSchema, addNode } from '../../validator/Validator';
+import { Template } from 'jsoneditor';
+
+const generateCaption = (key: string) => {
```
Comment (malindakpt) [inline src/components/JsonEditor/JsonEditor.tsx:8]: I feel, display the key as it is also ok

**Review by malindakpt — APPROVED** [review]
Looks good for now

File: src/components/JsonEditor/JsonEditor.tsx (hunk 2 above)
Comment (gayanDeegala) [reply src/components/JsonEditor/JsonEditor.tsx:8]: This function does not use for now

**Pull request merged**